notes/v1.5.5.md

```diff
@@ -0,0 +1,146 @@
+# bloop `v1.5.5`
+
+Bloop v1.5.5 is a bugfix release.
```

Comment:
I'd also add something like the following

> One notable change this release for anyone using the `bloop-config` artifact is that it's not longer contained inside of this repo or versioned along with it. You can find its new location in [`scalacenter/bloop-config`](https://github.com/scalacenter/bloop-config). The latest version of `bloop-config` is still 1.5.5 and you'll continue to use it as your normally have.

Reply:
Added! Good idea!